python/servo/devtools_tests/sources/test.html

```diff
@@ -8,4 +8,4 @@
     import module from "./module.js";
     console.log("inline module");
 </script>
-<script src="https://servo.org/js/load-table.js"></script>
+<script src="http://127.0.0.1:10001/classic.js"></script>
```

Comment:
This seems like a footgun; do we need the full host and port or can we just use `src="classic.js"`?